### PR TITLE
My PR

### PR DESCRIPTION
**Pull request by SebastianO1717**

### CODE REVIEW

File: Extensions/Program.cs [hunk 1]
```diff
@@ -1,2 +1,46 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿/// <summary>
+/// Why extension methods? Extension methods enable you to "add" methods to existing types without creating a new derived type
+/// The extension methods are static methods, but they're called as if they were instance methods on the extended type. 
+/// 
+/// When using an extension method to extend a type whose source code you aren't in control of, you run the risk that a change in the implementation of the type will cause your extension method to break.
+/// 
+/// 
+/// </summary>asdfasdfsdafasdf
```
Comment (SebastianO1717) [inline Extensions/Program.cs:8]: add a space between

File: Extensions/Program.cs [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+/// 
+// New Comment my new change
+public interface FooInterface
```
Comment (Sebastian1712) [inline Extensions/Program.cs:11]: Nice interface

Comment (Sebastian1712) [reply Extensions/Program.cs:11]: Thanks!

File: Extensions/Program.cs [hunk 3]
```diff
@@ -1,2 +1,46 @@
...
+{
+    string Name { get; set; }
+    string GetName();
+}
+
+class FooA : FooInterface
```
Comment (Sebastian1712) [inline Extensions/Program.cs:17]: I like the name

Comment (SebastianO1717) [reply Extensions/Program.cs:17]: thanks

Comment (SebastianO1717) [reply Extensions/Program.cs:17]: ill keep it

File: Extensions/Program.cs (hunk 1 above)
Comment (SebastianO1717) [reply Extensions/Program.cs:8]: will do

File: Extensions/Program.cs (hunk 2 above)
Comment (SebastianO1717) [reply Extensions/Program.cs:11]: wow

**Pull request closed without merging**